src/mono/mono.proj

```diff
@@ -50,6 +50,13 @@
     <MonoCxxCompiler Condition="'$(MonoCCompiler)' == 'gcc'">g++</MonoCxxCompiler>
   </PropertyGroup>
 
+  <!-- default metadata update support for specific platforms -->
+  <PropertyGroup>
+    <MonoMetadataUpdate Condition="'$(MonoMetadataUpdate)' == '' and ('$(TargetsAndroid)' == 'true' or '$(TargetsBrowser)' == 'true')">true</MonoMetadataUpdate>
+
```

Comment:
Shouldn't this be enabled in debug mode only ?

Reply:
Debug of the runtime? No.

It should be in debug builds of the end-user app / workload.  But that doesn't exist yet in our packaging.

But we're going to turn it on for these two workloads for a few previews so that the downstream teams can use it from the existing runtime packs.